bigchaindb/backend/rethinkdb/admin.py

```diff
@@ -0,0 +1,53 @@
+"""Database configuration functions."""
+import logging
+
+import rethinkdb as r
+
+from bigchaindb.backend import admin
+from bigchaindb.backend.schema import TABLES
+from bigchaindb.backend.exceptions import DatabaseOpFailedError
+from bigchaindb.backend.utils import module_dispatch_registrar
+from bigchaindb.backend.rethinkdb.connection import RethinkDBConnection
+
+logger = logging.getLogger(__name__)
+
+register_admin = module_dispatch_registrar(admin)
+
+
+@register_admin(RethinkDBConnection)
+def config(connection, *, table):
```

Comment:
Good point. I added it for two reasons:

1. To have uniform function signatures. They all have `connection` as a first positional argument, followed by the required and/or keyword arguments.
2.  it made it easier and quicker to write the test via parametrization.

I must admit I was considerably influenced by point 2.

If you prefer, I can make `table` a positional argument, and adjust the tests accordingly.